static/js/settings_org.js

```diff
@@ -1112,6 +1112,38 @@ exports.build_page = function () {
             },
         });
     });
+
+    function update_export_status(status_text, is_error) {
+        var export_status = $('#export_status');
+        if (is_error) {
+            export_status.removeClass('alert-success').addClass('alert-danger');
+        } else {
+            export_status.removeClass('alert-danger').addClass('alert-success');
+        }
+
+        export_status.find('.export_status_text').text(status_text);
+        export_status.show();
+
+        $(".close-export-status").on('click', function () {
+            $('#export_status').hide();
+        });
+    }
+
+    $("#export-data").on('click', function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+
+        channel.post({
+            url: '/json/export/realm',
+            success: function (response) {
+                update_export_status(i18n.t(response.msg), false);
+            },
+            error: function (response) {
+                var error_message = "Oops! " + response.responseJSON.msg;
+                update_export_status(i18n.t(error_message), true);
```

Comment:
Just wondering why didn't we used `ui_report.success` and `ui_report.error` directly over here, it would make the code more understandable and synchronized with the other settings. Moreover, I don't think "Oops!" should be the error message over here, it should be something more clear and precise.

Reply:
> Just wondering why didn't we used ui_report.success and ui_report.error

Great to know, thanks! I wasn't aware.

> Moreover, I don't think "Oops!" should be the error message over here

Certainly, I was skeptical of this being acceptable.  Just wanted to wait for insight from a reviewer :) Thanks @pragatiagrawal31